src/app/beer_garden/events/handlers.py

```diff
@@ -49,25 +53,53 @@ def downstream_callbacks(event: Event) -> None:
     """
     if event.garden != beer_garden.config.get("garden.name"):
         if event.error:
-            logger.error(f"Downstream error event ({event}): {event.error_message}")
+            logger.error(
+                f"Downstream error event ({event} : {event.payload_type}: {event.payload}): {event.error_message}"
+            )
             return
 
-        try:
-            if event.name in (Events.REQUEST_CREATED.name, Events.SYSTEM_CREATED.name):
-                db.create(event.payload)
+        if not event.payload_type:
+            logger.error(
+                f"Unable to process event ({event} : {event.payload_type}: {event.payload}): No Payload Type"
+            )
+            return
```

Comment:
Is there a reason we need this here? It seems reasonable that we could have an event without a payload at all (if we were only publishing fact-of occurrence or something).

Reply:
I moved it down to the statements where I require that field. So now if we publish an update/delete event without the model type, we will throw an error stating that we can't process something we don't know what it is.